e2e_tests/tests/per_provider/normal_tests/asym_sign_verify.rs

```diff
@@ -233,9 +336,37 @@ fn sign_hash_bad_format() -> Result<()> {
     Ok(())
 }
 
+#[cfg(feature = "cryptoauthlib-provider")]
 #[test]
-fn simple_verify_hash() -> Result<()> {
-    let key_name = String::from("simple_verify_hash");
+fn sign_hash_bad_format_ecdsa_sha256() -> Result<()> {
+    let key_name = String::from("sign_hash_bad_format_ecdsa_sha256");
+    let mut client = TestClient::new();
+    if !client.is_operation_supported(Opcode::PsaGenerateKey) {
+        return Ok(());
+    }
+    if !client.is_operation_supported(Opcode::PsaSignHash) {
+        return Ok(());
+    }
+
+    let hash1 = vec![0xEE; 31];
+    let hash2 = vec![0xBB; 33];
+
+    client.generate_ecc_key_pair_secpr1_ecdsa_sha256(key_name.clone())?;
+    // Issue - mbed-crypto-provider returns Ok() here:
```

Comment:
This seems to be a potential issue with Mbed Crypto - I've raised an issue in the interface (https://github.com/parallaxsecond/parsec-interface-rs/issues/107) to enforce this more consistently across the providers, for now it's ok to leave as is.

Reply:
I planned to raise the issue after this PR is merged - thank you for doing this.
And the input parameters are properly checked in Mbed Crypto when RSA is used.